flink/v1.14/flink/src/main/java/org/apache/iceberg/flink/sink/IcebergFilesCommitter.java

```diff
@@ -129,6 +135,10 @@ public void initializeState(StateInitializationContext context) throws Exception
     Preconditions.checkArgument(maxContinuousEmptyCommits > 0,
         MAX_CONTINUOUS_EMPTY_COMMITS + " must be positive");
 
+    int manifestsScanParallelism = PropertyUtil.propertyAsInt(
+        table.properties(), MANIFESTS_SCAN_PARALLELISM, ThreadPools.WORKER_THREAD_POOL_SIZE);
+    this.workerPool = ThreadPools.newWorkerPool("iceberg-files-committer-worker-pool", manifestsScanParallelism);
```

Comment:
configured from `table#properties` like the `MAX_CONTINUOUS_EMPTY_COMMITS` configuration  to make it simple for now. Add another ways if it's needed in the future.